dvc/remote/local.py

```diff
@@ -710,3 +720,22 @@ def _cache_metadata_changed(self):
             return not (mtime == cached_mtime and size == cached_size)
 
         return True
+
+    def _log_missing_caches(self, checksum_info_dict):
+        missing_caches = [
+            (md5, info)
+            for md5, info in checksum_info_dict.items()
+            if info["status"] == STATUS_MISSING
+        ]
+        if missing_caches:
+            missing_desc = "".join(
+                [
+                    "\nname: {}, md5: {}".format(info["name"], md5)
+                    for md5, info in missing_caches
+                ]
+            )
+            msg = (
+                "Some of the caches do not exist neither locally "
```

Comment:
It seems like "caches" is misleading. Maybe "cache files" would be better to avoid confusion with multiple cache locations? Also, we don't really use "objects" in relation to our cache yet, it is a git terminology.